creator-node/scripts/dev-server.sh

```diff
@@ -2,7 +2,7 @@
 set -o xtrace
 set -e
 
-link_libs=true
+link_libs=false
```

Comment:
do we want this? i think it's kinda nice to have

Reply:
i can add this back in... but the tradeoff is for local dev we need to rebuild libs locally with node 14 for the link to work

Reply:
can we just have everyone nvm use > 14? maybe im missing something

Reply:
not all the services have been built using node 14. we'd have to update content node, identity and libs to blanket update to node 14 and i wasn't sure if we wanted to do everything in one giant update. instead i was thinking we break it up by service, test functionality with qa runs and let it soak on prod to give us more of a chance to isolate regressions.

that's why i was thinking if someone needed libs they could link and npm install in libs by hand until we do all the services

Reply:
i doubt too much stuff would break, but cool 👍 sounds good